SharedPackages/DataBrokerProtectionCore/Tests/DataBrokerProtectionCoreTests/BrokerProfileJobActionTests.swift

```diff
@@ -467,78 +467,6 @@ final class BrokerProfileJobActionTests: XCTestCase {
-            stageDurationCalculator: MockStageDurationCalculator(),
-            pixelHandler: MockPixelHandler(),
-            executionConfig: BrokerJobExecutionConfig(),
-            shouldRunNextStep: { true }
-        )
-
-        XCTAssertEqual(sut.retriesCountOnError, 0)
-        await sut.runNextAction(expectationAction)
-        XCTAssertEqual(sut.retriesCountOnError, 1)
-    }
-
-    func testWhenExpectationActionRunsDuringOptOut_thenRetriesCountIsSetToThree() async {
-        let expectationAction = ExpectationAction(id: "1", actionType: .expectation, expectations: [Item](), dataSource: nil, actions: nil)
-        let sut = BrokerProfileOptOutSubJobWebRunner(
-            privacyConfig: PrivacyConfigurationManagingMock(),
-            prefs: ContentScopeProperties.mock,
-            context: BrokerProfileQueryData.mock(),
-            emailConfirmationDataService: emailConfirmationDataService,
-            captchaService: captchaService,
-            featureFlagger: MockDBPFeatureFlagger(),
-            operationAwaitTime: 0,
-            stageCalculator: MockStageDurationCalculator(),
-            pixelHandler: pixelHandler,
-            executionConfig: BrokerJobExecutionConfig(),
-            actionsHandlerMode: .optOut,
-            shouldRunNextStep: { true }
-        )
-
-        XCTAssertEqual(sut.retriesCountOnError, 3)
-        await sut.runNextAction(expectationAction)
-        XCTAssertEqual(sut.retriesCountOnError, 3)
-    }
-
-    func testWhenExpectationActionFailsDuringScan_thenRetryOnce() async {
-        let expectationAction = ExpectationAction(id: "1", actionType: .expectation, expectations: [Item](), dataSource: nil, actions: nil)
-        let step = Step(type: .scan, actions: [expectationAction])
-        let sut = BrokerProfileScanSubJobWebRunner(
-            privacyConfig: PrivacyConfigurationManagingMock(),
-            prefs: ContentScopeProperties.mock,
-            context: BrokerProfileQueryData.mock(with: [Step(type: .scan, actions: [])]),
-            emailConfirmationDataService: emailConfirmationDataService,
-            captchaService: captchaService,
-            featureFlagger: MockDBPFeatureFlagger(),
-            stageDurationCalculator: MockStageDurationCalculator(),
-            pixelHandler: MockPixelHandler(),
-            executionConfig: BrokerJobExecutionConfig(),
-            shouldRunNextStep: { true }
-        )
-        sut.webViewHandler = webViewHandler
-        sut.actionsHandler = ActionsHandler.forScan(step)
-
-        await sut.runNextAction(expectationAction)
-        XCTAssertEqual(sut.retriesCountOnError, 1)
-
-        _ = sut.actionsHandler?.nextAction()
-
-        await sut.onError(error: DataBrokerProtectionError.httpError(code: 429))
-        XCTAssertEqual(sut.retriesCountOnError, 0)
-
-        await sut.onError(error: DataBrokerProtectionError.httpError(code: 429))
-        XCTAssertTrue(webViewHandler.wasFinishCalled)
-    }
-
```

Comment:
These test cases no longer make sense due to the way the sub job runners are implemented.